setup.py

```diff
@@ -59,8 +59,6 @@ def pip(filename):
 exec(compile(open('flask_restplus/__about__.py').read(), 'flask_restplus/__about__.py', 'exec'))
 
 install_requires = pip('install')
-if sys.version_info < (3, 4):
-    install_requires += ['enum34']
```

Comment:
Does the `pip()` function not catch this? Weird...

Reply:
Why should pip() catch it? There is no reference between the method and these two lines.

Reply:
The pip function call in line 61 parses the `requirements/install.pip`, in which you've added `enum34` as a requirement.

Reply:
🤦‍♂️ I just realized these are removal lines and not additions... Never mind 😅